src/main/java/com/api/pedidosApi/services/CategoriaService.java

```diff
@@ -16,14 +15,12 @@ public class CategoriaService {
 
     public List<Categoria> FindAll() {
        return categoriaRepository.findAll();
-      // return ResponseEntity.ok().body(categoriaRepository.findAll())
     }
 
      public Categoria FindOne(Integer id) {
-        //return categoriaRepository.findById(id)
-        //        .orElseThrow(() -> new ErrosNotFoundException("Categoria not found with ID: " + id));
        Optional<Categoria> obj = categoriaRepository.findById(id);
-       return obj.orElseThrow(() -> new ErrosNotFoundException("Objeto não encontrado! Id: " + id + ", Tipo: " + Categoria.class.getName()));
+      return obj.orElseThrow(() -> new ObjectNotFoundException(
+              "Objeto não encontrado! Id: " + id + ", Tipo: " + Categoria.class.getName()));
     }
 
     public Categoria Insert(Categoria obj){
```

Comment:
Nomes de métodos devem iniciar com letra minúscula
```suggestion
    public Categoria insert(Categoria obj){
```